Makefile

```diff
@@ -0,0 +1,27 @@
+CC = gcc
+CFLAGS = -Wall -g -I/usr/include/pipewire-0.3 -I/usr/include/spa-0.2
+LDFLAGS = -lncurses -lpipewire-0.3 -lm
```

Comment:
I believe this should be -lncursesw for wide-character support. When I install it using the Makefile all the `·`(middle dots) get replaced by `m`s.

Reply:
I have fix the `-lncursesw`.